addons/website/static/src/snippets/s_popup/000.js

```diff
@@ -111,23 +111,5 @@ const PopupWidget = publicWidget.Widget.extend({
 
 publicWidget.registry.popup = PopupWidget;
 
-// Prevent bootstrap to prevent scrolling and to add the strange body
-// padding-right they add if the popup does not use a backdrop (especially
-// important for default cookie bar).
-const _baseSetScrollbar = $.fn.modal.Constructor.prototype._setScrollbar;
-$.fn.modal.Constructor.prototype._setScrollbar = function () {
-    if (this._element.classList.contains('s_popup_no_backdrop')) {
-        return;
-    }
-    return _baseSetScrollbar.apply(this, ...arguments);
-};
-const _baseGetScrollbarWidth = $.fn.modal.Constructor.prototype._getScrollbarWidth;
-$.fn.modal.Constructor.prototype._getScrollbarWidth = function () {
-    if (this._element.classList.contains('s_popup_no_backdrop')) {
-        return 0;
-    }
-    return _baseGetScrollbarWidth.apply(this, ...arguments);
-};
```

Comment:
This code is there for a reason as explained by the comment just above it. If you remove it, you prevent scrolling website page when all cookie bar are opened (even the small discrete one at the bottom). And in general: you remove our feature of popup without backdrop.